### PR TITLE
Jail Entity

### PR DESCRIPTION
**Pull request by Winterequiem**

### CODE REVIEW

File: src/main/java/tutorial/Jail.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+// roll double thrice consecutively
+// go to jail chance card (if passed go to land on card chest, collect go money, else go directly to jail)
+// land on jail (visiting, no penalty)
+
+import java.util.ArrayList;
+
+public class Jail {
+    Player player;
+    private int turnsInJail;
+    private boolean inJail;
```
Comment (adamjvalentini) [inline src/main/java/tutorial/Jail.java:12]: Should the inJail Variable be public? - so that it can be checked in other classes if a player is in jail

**Review by adamjvalentini — COMMENTED** [review]
Overall it looks really good!

File: src/main/java/tutorial/Jail.java (hunk 1 above)
Comment (Winterequiem) [reply src/main/java/tutorial/Jail.java:12]: there is a getter method for inJail.

**Review by adamjvalentini — DISMISSED** [review]

**Review by adamjvalentini — APPROVED** [review]

**Review by raelws — APPROVED** [review]
Looks good overall

**Review by pujiacheng — APPROVED** [review]

**Pull request merged**